### PR TITLE
PRM Automated Test - LPS-172957

### PR DESCRIPTION
**Pull request by marinaasant0s**
[LPS-172957](https://issues.liferay.com/browse/LPS-172957)

### CODE REVIEW

**Comment by marinaasant0s** [comment]
ci:test:sf

**Comment by marinaasant0s** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/macros/PRMMarketing.macro [hunk 1]
```diff
@@ -5,4 +5,89 @@ definition {
 		Click(locator1 = "PRMMarketing#APPLY_FOR_FUNDS");
 	}
 
+	@summary = "Verify the activity fills made by the user"
```
Comment (AfonsoHSantos) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/macros/PRMMarketing.macro:8]: Hi @marinaasant0s, could you please update this summary to: "View the activity information on MDF Review Page"?

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/macros/PRMMarketing.macro [hunk 2]
```diff
@@ -5,4 +5,89 @@ definition {
...
+			key_fieldValue = "${budget}",
+			locator1 = "PRMMDFRequest#SUMMARY_FIELD");
+
+		AssertElementPresent(
+			key_fieldName= "Is a lead list an outcome of this activity?",
+			key_fieldValue = "${leadList}",
+			locator1 = "PRMMDFRequest#SUMMARY_FIELD");
+	}
+	
+	@summary = "Verify the fills done by the user on the Campaign Information"
```
Comment (AfonsoHSantos) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/macros/PRMMarketing.macro:55]: Hi @marinaasant0s, could you please update this summary to: "View the goals information on MDF Review Page

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/macros/PRMMarketing.macro [hunk 3]
```diff
@@ -5,4 +5,89 @@ definition {
...
+	macro viewMDFContent {
```
Comment (AfonsoHSantos) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/macros/PRMMarketing.macro:56]: Hi @marinaasant0s, could you please rename this macro to: "viewMDFGoals"?

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase [hunk 4]
```diff
@@ -119,6 +119,89 @@ definition {
...
+			var activityName = "Activity Name 1";
+			var budgetList = "Sponsorship Fee-2000";
+			var expense = "Sponsorship Fee";
+			var budget = "$2,000.00";
+			var leadList = "No";
+			var tactic = "Other";
+			var typeOfActivity = "Miscellaneous Marketing";
+			var startDate = DateUtil.getFormattedDate("+1", "M/dd/yyyy");
+			var endDate = DateUtil.getFormattedDate("+2", "M/dd/yyyy");
+		}
```
Comment (AfonsoHSantos) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase:146]: Hi @marinaasant0s, isn't necessary a task for this variable definitions. It should be before the previous task also.

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase [hunk 5]
```diff
@@ -119,6 +119,89 @@ definition {
...
+
+		task ("Then you get the appropriate results") {
```
Comment (AfonsoHSantos) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase:148]: Hi @marinaasant0s, could you please update this task to: "When I fill all fields from MDF Request"

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase [hunk 6]
```diff
@@ -119,6 +119,89 @@ definition {
...
+				tactic = "${tactic}",
+				targetOfLeads = "${targetOfLeads}",
+				typeOfActivity = "${typeOfActivity}");
+
+			Button.click(button = "Continue");
+
+			Button.click(button = "Continue");
+		}
+
+		task ("And verify that all the filled in fields are present") {
```
Comment (AfonsoHSantos) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase:177]: Hi @marinaasant0s, could you please update this task to: "Then all inputted information are displayed on Review Page"

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase [hunk 7]
```diff
@@ -119,6 +119,89 @@ definition {
...
+				endDate = "${endDate}",
+				expense = "${expense}",
+				leadList = "${leadList}",
+				startDate = "${startDate}",
+				tactic = "${tactic}",
+				typeOfActivity = "${typeOfActivity}");
+
+			ScrollWebElementIntoView(locator1 = "Button#SUBMIT");
+
+			Button.click(button = "Submit");
```
Comment (AfonsoHSantos) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase:201]: Hi @marinaasant0s, isn't necessary to submit the MDF

**Review by AfonsoHSantos — CHANGES_REQUESTED** [review]

**Comment by marinaasant0s** [comment]
ci:test:sf

**Comment by AfonsoHSantos** [comment]
Hi @marinaasant0s, thanks for working on it. The code looks good to me! I'm forwarding it.

giphy

**Comment by AfonsoHSantos** [comment]
ci:forward

**Pull request closed without merging**